src/sql/plot.py

```diff
@@ -607,6 +618,19 @@ def _histogram_stacked(
     return data
 
 
+@modify_exceptions
+def _filter_aggregate(filter_queries):
+    """Return a single filter query based on multiple queries"""
+    final_filter = ""
+    for query in filter_queries:
+        if query is not None:
+            if query == filter_queries[0]:
+                final_filter += f"WHERE {query}"
+                continue
+            final_filter += f" AND {query}"
```

Comment:
Let's use format instead +.

`final_filter += f" AND {query}"` -> `final_filter = f"{final_filter} AND {query}"`

Reply:
Sure! making the changes